### ISSUE TITLE
simulating a 9 point dither

### ISSUE THREAD

**Issue by havok2063** [sdss/lvmdatasimulator#50]
I think it would be useful to simulate a full 9 point dither, or a general X-point dither, with a single simulation run.  Currently to do this, you need to re-run the simulator 9 times shifting the position of the IFU bundle in the field by hand.  A useful input parameter would be a number for tuning the relative positions of the bundles.  For example `0` would align all 9 IFUs on top of each other at the same location, complete overlap, and `1` would align all 9 IFUs in a contiguous tiling, with no overlap between the bundles.  Intermediate values could scale out the bundles in incremental steps.    

This would make simulating multiple observations much easier, and could help with designing image and cube reconstruction modules.

**Comment by cloud182** [comment]
Hi, thanks @havok2063 for your comment. We know that simulating a dithering pattern automatically would be something interesting to do with the simulator, we have it on our list of things to add to the software. Unfortunately, implementing it is not really straightforward. It would require some significant changes in how the spectra are managed inside the code and we had some other higher priorities (the 2D simulations) before moving to this. As soon as we have a working version of the 2D module, we will move to all the other improvements we are planning. 

To summarize, we cannot do it immediately, but we have it on our list and will try to tackle this as soon as possible!

**Comment by havok2063** [comment]
@cloud182 that makes sense.  I'm sure there are higher priorities for science and pipeline testing.  I'm interested in this feature so I can generate a decent chunk of simulated data and begin development on a prototype interface for visualizing and selecting spaxels.   Hopefully it can implemented at some point but in the meantime I can work on an interim solution.
